app/scripts/components/common/blocks/block-map.tsx

```diff
@@ -361,10 +373,18 @@ function MapBlock(props: MapBlockProps) {
                 />
               )}
           </Compare>
-        )}
+        )} */}
       </Map>
     </Carto>
   );
 }
 
+export function MapBlockWithProvider(props: MapBlockProps) {
+  return (
+    <ReactQueryProvider>
```

Comment:
I know you're actively working on this branch right now, but I wanted to share some thoughts in case they might be helpful.


If I understand React Query correctly, the provider allows us to cache data so we don't have to fetch the same information repeatedly. If each map is wrapped with its own separate Provider, however, this cached data won't be shared across maps. 

One idea could be to export all the providers that UI requires from index.ts, and provide guidance for users on how to wrap their root layout with the necessary Providers. We already do something similar for ThemeProvider in our UI library (see [link](https://github.com/NASA-IMPACT/veda-ui/pull/1046/files#diff-df6782dfe8efb20003aa046424999792c0fdfa4472bc6b8f86a487b344ee0f6cR18)).